src/ResultWriter/DefaultResultWriter.php

```diff
@@ -78,7 +78,19 @@ public function writeToCsv(
         $incFetchingColMaxValue = $this->lastRow ?
             $this->getIncrementalFetchingValueFromLastRow($exportConfig) :
             $this->getIncrementalFetchingValueFromState($exportConfig);
-        return new ExportResult($csvFilePath, $this->rowsCount, $result->getMetadata(), $incFetchingColMaxValue);
+        return new ExportResult(
+            $csvFilePath,
+            $this->rowsCount,
+            $result->getMetadata(),
+            $this->hasCsvHeader($exportConfig),
+            $incFetchingColMaxValue
+        );
+    }
+
+    protected function hasCsvHeader(ExportConfig $exportConfig): bool
```

Comment:
Ohladom PDO/ODBC (default writer) som to dal do osobitnej metody, aby sa to dalo pripadne upravit, ak bude potrebne.